features/step_definitions/request_steps.ts

```diff
@@ -263,7 +256,7 @@ Then(
   /the response "([^"]+)" is equal to (.*)/,
   function (this: World, responsePath: string, value: string) {
     expect(pathLookup(this.response, responsePath)).to.deep.equal(
-      JSON.parse(value.templated(this.fixtures))
+      remapToAttributeName(JSON.parse(value.templated(this.fixtures)))
```

Comment:
Does that lose the fact that we selectively rename attributes in the serializer? Eg for additionalProperties?